src/server.c

```diff
@@ -3232,6 +3232,31 @@ void propagatePendingCommands() {
     redisOpArrayFree(&server.also_propagate);
 }
 
+/* Performs operations that should be performed after an execution unit ends.
+ * Execution unit is a code that should be done atomically.
```

Comment:
Are logical units = execution units? It seems like this paragraph is describing one thing, so maybe just be consistent. Either that, or I'm not sure what the difference is.

Reply:
Fixed to execution unit